blog/20190318-mate-1-22-released.md

```diff
@@ -0,0 +1,2140 @@
+<!--
+.. link:
+.. description:
+.. tags: Releases
+.. date: 2019-03-18 09:00:00
+.. title: MATE 1.22 released
+.. slug: 2019-03-18-mate-1-22-released
+.. author: Victor Kareh
+-->
+
+After about a year of development, the MATE Desktop team have finally released MATE 1.22. A big thank you to all contributors who helped to make this happen.
+
+![about|(/assets/img/blog/about-mate-1.22.png)
```

Comment:
The image doesn't show up here on a local build! :confused:
Shouldn't it be ``![about](/assets/img/blog/about-mate-1.20.png)``?